data/tutorials/gs_00_up_and_running.md

```diff
@@ -79,20 +90,61 @@ available:
 sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)
 ```
 
-### For Windows
+### Installation for Windows
+
+In this section we'll describe using the new [Diskuv OCaml](https://github.com/diskuv/dkml-installer-ocaml#readme) ("DKML") Windows installer. Expect to see another officially supported Windows installation provided directly by Opam in the coming months; it will be compatible with your DKML installation.
```

Comment:
```suggestion
In this section, we'll describe using the new [Diskuv OCaml](https://github.com/diskuv/dkml-installer-ocaml#readme) ("DKML") Windows installer. Expect to see another officially-supported Windows installation provided directly by opam in the coming months; it will be compatible with your DKML installation.
```